src/components/NewContactForm.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+export default function NewContactForm ({contacts, setContacts}) {
+    
+  const [newContact, setNewContact] = useState({
+    name: '',
+    email: '',
+    phone_number: '',
+    image_url: '',
+    id: Math.round(Math.random()*10000000000)
+  })
+
+  newContact.proptypes = {
```

Comment:
I'm not sure if this works, but ideally you'd use `PropTypes` on a component to ensure it's props are receiving the right types. So in this case you'd ensure that `NewContactForm`'s `contacts` and `setContacts` are what you expect them to be.